hipogram/posts/templates/post_list.html

```diff
@@ -2,18 +2,28 @@
 
 {% block content %}
 
+  {% if messages %}
+  <ul class="messages">
+      {% for message in messages %}
+      <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+      {% endfor %}
+  </ul>
+  {% endif %}
```

Comment:
Would be nice to use some styling but it's okay.

Reply:
Also, you can move this to `base.html`, adding it with include tag may be even nicer + cleaner.